components/server/src/bitbucket/bitbucket-repository-provider.ts

```diff
@@ -59,24 +59,39 @@ export class BitbucketRepositoryProvider implements RepositoryProvider {
     async getBranches(user: User, owner: string, repo: string): Promise<Branch[]> {
         const branches: Branch[] = [];
         const api = await this.apiFactory.create(user);
-        const response = await api.repositories.listBranches({
-            workspace: owner,
-            repo_slug: repo,
-            sort: "target.date",
-        });
 
-        for (const branch of response.data.values!) {
-            branches.push({
-                htmlUrl: branch.links?.html?.href!,
-                name: branch.name!,
-                commit: {
-                    sha: branch.target?.hash!,
-                    author: branch.target?.author?.user?.display_name!,
-                    authorAvatarUrl: branch.target?.author?.user?.links?.avatar?.href,
-                    authorDate: branch.target?.date!,
-                    commitMessage: branch.target?.message || "missing commit message",
-                },
+        // Handle pagination.
+        let nextPage: number | undefined = 1;
+        let isMoreDataAvailable: boolean = true;
```

Comment:
I think we can safely remove the type annotations here
```suggestion
        let nextPage = 1;
        let isMoreDataAvailable = true;
```